vllm/utils.py

```diff
@@ -58,7 +58,9 @@ def in_wsl() -> bool:
 
 
 def get_ip() -> str:
-    return socket.gethostbyname(socket.gethostname())
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    s.connect(("8.8.8.8", 80))  # Doesn't need to be reachable
```

Comment:
what happen if it is not reachable? will it timeout/error?

Reply:
i tried some other non-reachable addresses and this still works